MODULE.bazel

```diff
@@ -17,10 +18,16 @@ bazel_dep(
 
 bazel_dep(
     name = "stardoc",
-    version = "0.6.2",
+    version = "0.7.1",
```

Comment:
Can we get away without upgrading this? I want to save the change to the docs where it uses `@rules_apple` for when we drop Bazel. I was thinking we could add the flag to ignore the empty globs to the Doc job instead for now.

Reply:
Unfortunately not:

```
ERROR: Traceback (most recent call last):
        File "/private/var/tmp/_bazel_lpadron/d8d9324fa7fd2d11d5178d8cf2834c55/external/stardoc+/stardoc/BUILD", line 72, column 16, in <toplevel>
                srcs = glob(["*.jar"]),
```

Which was fixed in 0.7.1 it seems

Reply:
Oh i see:

>  I was thinking we could add the flag to ignore the empty globs to the Doc job instead for now.

Yeah lets do that for stardoc

Reply:
> I want to save the change to the docs where it uses @rules_apple for when we drop Bazel.

Even with stardoc 0.6.2 using last_green Bazel the `load("@rules_apple")` stuff is added to all docs.

Reply:
Hmm. I only saw that when using the newer stardoc.

Reply:
`protobuf` has a non-dev dependency on `stardoc` 🙃

Reply:
Fixed the stardoc upgrade issue, it was being brought in by protobuf:

```starlark
# TODO: Remove override when a protobuf release that marks `stardoc` as a
# dev_dependency is available, until then it's upgrading our stardoc version
# so override it here.
single_version_override(
    module_name = "stardoc",
    version = "0.6.2",
)
```